libs/components/src/lib/text-area/text-area.spec.ts

```diff
@@ -448,6 +448,61 @@ describe('vwc-text-area', () => {
 		});
 	});
 
+	describe('forced error', function () {
+		const forcedErrorMessage = 'BAD!';
+
+		it('should force the input in custom error mode', async function () {
+			element.errorText = forcedErrorMessage;
+			await elementUpdated(element);
+			expect(element.validationMessage).toBe(forcedErrorMessage);
+			expect(element.validity.valid).toBeFalsy();
+		});
+
+		it('should add the error class', async function () {
+			element.errorText = forcedErrorMessage;
+			await elementUpdated(element);
+			expect(getBaseElement(element)
+				.classList
+				.contains('error'))
+				.toEqual(true);
+		});
+
+		it('should display the given error message', async function () {
+			element.errorText = forcedErrorMessage;
+			await elementUpdated(element);
+			const errorElement = element.shadowRoot?.querySelector('.error-message');
+			expect(errorElement !== null).toBeTruthy();
+		});
+
+		it('should replace/restore the current error state, if any, when set/removed', async function () {
+			let initialErrorMessage = '';
+
+			expect(element.validationMessage).toBe('');
+			expect(element.validity.valid).toBeTruthy();
+
+			element.required = true;
+			setToBlurred();
+			await elementUpdated(element);
+			initialErrorMessage = element.validationMessage;
+
+			expect(initialErrorMessage).not.toBe('');
+			expect(initialErrorMessage).not.toBe(forcedErrorMessage);
+			expect(element.validity.valid).toBeFalsy();
+
+			element.errorText = forcedErrorMessage;
+			await elementUpdated(element);
+
+			expect(element.validationMessage).toBe(forcedErrorMessage);
+			expect(element.validity.valid).toBeFalsy();
+
+			element.errorText = '';
+			await elementUpdated(element);
+
+			expect(element.validationMessage).toBe(initialErrorMessage);
+			expect(element.validity.valid).toBeFalsy();
+		});
+	});
```

Comment:
Something in the spirit of:
```suggestion
		it('should replace the current error state when set', function () {
		element.required = true;
		setToBlurred();
		element.errorText = forcedErrorMessage;
		await elementUpdated(element);

		expect(element.validationMessage).toBe(forcedErrorMessage);
	});
	
	it('should restore the current error state when removed', async function () {
		element.required = true;
		setToBlurred();
		await elementUpdated(element);
		const initialErrorMessage = element.validationMessage;
		element.errorText = forcedErrorMessage;
		await elementUpdated(element);

		element.errorText = '';
		expect(element.validationMessage).toBe(initialErrorMessage);
	});
```